### PR TITLE
Add postgres command with path to config file to docker-compose staging

### PR DESCRIPTION
**Pull request by dchiller**
Simultaneous work on #1286 and #1293 meant the staging and production versions of `docker-compose.yml` had different commands. This fixes the command in the staging version with an argument pointing to the location of the `postgres.conf` file in the `postgres` container.

### CODE REVIEW

File: docker-compose.yml.staging [hunk 1]
```diff
@@ -37,6 +37,7 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data/
     restart: always
+    command: postgres -c 'config_file=/etc/postgresql/postgresql.conf'
```
Comment (jacobdgm) [inline docker-compose.yml.staging:40]: Please add this line to `docker-compose.yml.development` too!

**Review by jacobdgm — CHANGES_REQUESTED** [review]

File: docker-compose.yml.staging (hunk 1 above)
Comment (dchiller) [reply docker-compose.yml.staging:40]: do we want that? Weren't these determined for the servers only?

Comment (jacobdgm) [reply docker-compose.yml.staging:40]: I think we want the configuration to be as similar as possible between develop, staging and production, don't we?

Comment (jacobdgm) [reply docker-compose.yml.staging:40]: I can see where you're coming from, though.

**Review by jacobdgm — APPROVED** [review]

**Pull request merged**